src/solc_0.5/Land/erc721/LandBaseTokenV3.sol

```diff
@@ -242,13 +242,13 @@ contract LandBaseTokenV3 is ERC721BaseTokenV2 {
         // Length of array is equal to number of 3x3 child quad a 24x24 quad can have
         Land[] memory quadMinted = new Land[](64);
         uint256 index;
-        uint256 landMinted;
+        uint256 numLandMinted;
```

Comment:
in the audit, they are saying that is the number of land transferred not minted

Reply:
but we are transferring the lands which are already minted in parent quad and this variable stores its count.
what other variable name do you suggest?